Gopkg.lock

```diff
@@ -2,14 +2,175 @@
 
 
 [[projects]]
+  branch = "master"
+  name = "github.com/Azure/go-ansiterm"
+  packages = [".","winterm"]
+  revision = "fa152c58bc15761d0200cb75fe958b89a9d4888e"
+
+[[projects]]
+  name = "github.com/Microsoft/go-winio"
```

Comment:
we use [golang/dep](https://github.com/golang/dep) to vendor dependencies currently. even though `go-winio` is only used when `GOOS=windows` or using `-tags=windows` which I don't think is a supported use case, the vendoring tool doesn't know that and imports any secondary dependencies (this one is from https://github.com/ilackarms/image-inspector/blob/dbae546537dcfcc4d9b55d66fb2646a1add513ea/vendor/github.com/containers/storage/pkg/system/filesys_windows.go)